website/public/sitemap.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Like with `graphql-sse`, would be great if you can gitignore this file since its generated on each deploy. Thanks!